supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -122,13 +155,15 @@
 [discrete]
 [[trusted-certificate-authority]]
 ==== image:images/yes.png[yes] trusted Certificate Authority (noun)
-*Description*: A "trusted Certificate Authority" is a third party that creates SSL certificates (CA certificates) used for authentication. It is not to be confused with self-signed certificates. Note the capitalization of Certificate Authority, commonly abbreviated as CA.
+*Description*: A trusted Certificate Authority (CA) is a third-party entity that creates TLS certificates, known as CA certificates, for authentication purposes. A trusted CA differs from a self-signed certificate in many ways. Note the capitalization of Certificate Authority, which is commonly abbreviated as CA.
```

Comment:
```suggestion
*Description*: A trusted Certificate Authority (CA) is a third-party entity that creates TLS certificates, known as CA certificates, for authentication purposes. A trusted CA is different from a self-signed certificate, being the second a signed certificate with its own private key instead of requesting it from a public or a private CA. Note the capitalization of Certificate Authority, which is commonly abbreviated as CA.
```

_in many ways_ is a bit ambiguous. I've suggested a sentence to explain the main difference. 

Other options:
a. Discard _A trusted CA is different ..._. 
b. Create a new entry for self-signed certificates and link to it.

WDYT?

Reply:
Thanks for the suggestion, @porueesq . I updated the content based on your suggestion. Please let me know your thoughts.

Reply:
Excellent. Thanks for the update. The MR is approved :partying_face: